Formula/swiftgen.rb

```diff
@@ -1,7 +1,7 @@
 class Swiftgen < Formula
   desc "Swift code generator for assets, storyboards, Localizable.strings, …"
   homepage "https://github.com/SwiftGen/SwiftGen"
-  url "https://github.com/SwiftGen/SwiftGen.git", :tag => "4.2.0"
+  url "https://github.com/SwiftGen/SwiftGen.git", :tag => "4.2.1"
```

Comment:
need the revision here too
```
  url "https://github.com/SwiftGen/SwiftGen.git",
      :tag => "4.2.1",
      :revision => "0ce335af8d3720e18b3553aede1205ae678de096"
```

I have no idea why it doesn't have it currently, since that should set up the fire alarms.